tests/foreman/ui/test_organization.py

```diff
@@ -376,41 +432,133 @@ def test_negative_update(self):
 
     @run_only_on('sat')
     @tier2
-    def test_positive_remove_domain(self):
+    def test_positive_create_with_subnet(self):
+        """Create Org with subnet.
+
+        :id: 6736cd82-a2b0-4fc0-a2bc-99c9f13464d7
+
+        :expectedresults: Org created with subnet.
+
+        :CaseLevel: Integration
+        """
+        with Session(self.browser) as session:
+            for subnet_name in generate_strings_list():
+                with self.subTest(subnet_name):
+                    # Create subnet using nailgun
+                    subnet = entities.Subnet(
+                        name=subnet_name,
+                        network=gen_ipaddr(ip3=True),
+                        mask='255.255.255.0'
+                    ).create()
+                    self.assert_org_creation_with(
+                        session, subnet, subnet_name, 'context.tab_subnets',
+                        subnets=[subnet_name]
+                    )
+
+    @run_only_on('sat')
+    @tier2
+    def test_positive_update_subnet(self):
+        """Add and Remove subnet by using organization name and subnet name.
+
+        :id: bc59bdeb-b538-4473-a096-e4de2454497d
+
+        :expectedresults: subnet is added then removed.
+
+        :CaseLevel: Integration
+        """
+        with Session(self.browser) as session:
+            org_name = gen_string('alpha')
+            make_org(session, org_name=org_name)
+            for subnet_name in generate_strings_list():
+
+                with self.subTest(subnet_name):
+                    # Create subnet using nailgun
+                    subnet = entities.Subnet(
+                        name=subnet_name,
+                        network=gen_ipaddr(ip3=True),
+                        mask='255.255.255.0',
+                    ).create()
+                    self.assert_org_update_with(
+                        org_name, session, subnet, subnet_name,
+                        'context.tab_subnets', subnets=[subnet_name]
+                    )
```

Comment:
@abalakh I totally agree on naming been more explicitly and most of time I struggle creating it. 

But the logic on all tests was very clear: Create Org, updating adding related entitiy, update removing entity. So I can't see how extracting this logic can be more difficult than CTLR+C  CTLR+V the entire logic over tests, so against DRY. Once you read helper assert for the first time, on first test, you can understand it for all other tests. The upside of this approach is, more than just having less code, update on assertion can be done in just one place. Just like know that we decided not to cache org on updates, I just changed that on helper. The first time I made the change to use cache I had to go over all tests and changing each on on the same way, risking forget some of them. In fact, this is exactly the problem with breaking

So I really like @SatelliteQE/quality-engineers opinion. I can revert it in case most think copy and paste is better, even without not at all agreeing to it ;)